routers/repo/attachment.go

```diff
@@ -44,7 +43,14 @@ func UploadAttachment(ctx *context.Context) {
 		buf = buf[:n]
 	}
 
-	err = upload.VerifyAllowedContentType(buf, strings.Split(setting.AttachmentAllowedTypes, ","))
+	var allowedTypes string
+	if ctx.Query("release") == "true" {
```

Comment:
Is this verified in any way, or can anyone just use `?release=true` to go around limitation?

Reply:
It's not really validated what type of upload it is. I think we need to either create a dedicated release-attachment endpoint or ensure that if this query is present, only releases can be created, not issue attachments.

Reply:
This is a hard one to crack because the endpoint is competely decoupled from the actual release POST, meaning it will first upload a attachment and then afterwards assign its ID to the release. I think the only easy way out is to create a secret token on the create/edit release pages and expect that back when verifying the attachment.